tendrl-monitoring-integration.spec

```diff
@@ -97,6 +99,7 @@ py.test -v tendrl/monitoring_integration/tests || :
 %dir %{_sysconfdir}/tendrl/monitoring-integration
 %doc README.rst
 %license LICENSE
+%config(noreplace) %{_sysconfdir}/firewalld/services/tendrl-monitoring-integration.xml
```

Comment:
Default firewalld xml config files should be packaged in `/usr/lib/firewalld/services/` (which would be something like `%{_prefix}/lib/firewalld/services/` for the purpose of a spec file) - so that admin could redefine them if necessary. See section "Place a file in the services directory in /usr/lib/firewalld" in http://www.firewalld.org/documentation/howto/add-a-service.html

Reply:
That said, doing this would also mean we need to depend on firewalld somehow. Let's check how other packages shipping firewalld service do it.

Reply:
I think this make sense. If we drop the xml file into /etc/firewalld folder and mess up the file is an issue. Instead keeping the file in /usr/lib/firewalld/services/ take care of this issue. Please have a look at http://www.firewalld.org/documentation/configuration/directories.html as well